projects/scripts/adi_project_xilinx.tcl

```diff
@@ -297,18 +304,25 @@ proc adi_project_files {project_name project_files} {
 #
 proc adi_project_run {project_name} {
 
+  global ad_project_dir
   global ADI_POWER_OPTIMIZATION
   global ADI_USE_OOC_SYNTHESIS
   global ADI_MAX_OOC_JOBS
 
+  if {![info exists ::env(ADI_PROJECT_DIR)]} {
+    set actual_project_name $project_name
+  } else {
+    set actual_project_name "$::env(ADI_PROJECT_DIR)${project_name}"
+  }
+
   if {$ADI_USE_OOC_SYNTHESIS == 1} {
     launch_runs -jobs $ADI_MAX_OOC_JOBS system_*_synth_1 synth_1
   } else {
     launch_runs synth_1
   }
   wait_on_run synth_1
   open_run synth_1
-  report_timing_summary -file timing_synth.log
+  report_timing_summary -file ${actual_project_name}_timing_synth.log
```

Comment:
Since those files are already in specific folder (the one created from parameters values), I think it's enough to just append path before the name without modifying the name itself, so to have: <project_path>/<folder_created_from_parameters>/timng_synth.log. 
This comment applies to all files like timing_impl.log, resource_utilization.csv, power_analisys.csv etc.